actions/xpubs/routes.go

```diff
@@ -20,6 +20,10 @@ func NewHandler(appConfig *config.AppConfig, services *config.AppServices) route
 		xpubGroup := router.Group("/xpub")
 		xpubGroup.GET("", action.get)
 		xpubGroup.PATCH("", action.update)
+
+		xpubGroup2 := router.Group("/users/current")
+		xpubGroup2.GET("", action.get2)
+		xpubGroup2.PATCH("", action.update2)
```

Comment:
Just as additional reminder, new API endpoints should be registered under `/api/v1` path

Reply:
changed